frontend/javascripts/types/schemas/dataset_view_configuration.schema.ts

```diff
@@ -90,12 +90,6 @@ export const defaultDatasetViewConfigurationWithoutNull: DatasetConfiguration =
   colorLayerOrder: [],
   nativelyRenderedLayerName: null,
 };
-export const defaultDatasetViewConfiguration = {
-  ...defaultDatasetViewConfigurationWithoutNull,
-  zoom: null,
-  position: null,
-  rotation: null,
-};
```

Comment:
Imo you changed logic here by removing this. In `dataset_view_configuration_defaults.ts` `defaultDatasetViewConfiguration` was used with the zoom, position & rotation null to fill in defaults. This is now missing in the default values. Therefore, the resulting object might not have 
```
zoom: null,
  position: null,
  rotation: null,
``` 
but instead those props might not be present which might be a logic change. Not sure whether this leads to any errors, but maybe we could prevent this or test for any errors this might create?

Reply:
> Therefore, the resulting object might not have ... but instead those props might not be present which might be a logic change.

yes, but what good do these null values do? :thinking: it's not clear what purpose they should serve. I couldn't find any problems with the changed code and also the tests in `dataset_view_configuration.spec.ts` succeed. the typechecker is also happy. therefore, I'd go with this simpler code now, unless you raise other concerns :)

Reply:
The potential prolem I see might occur in the backend in case some route expects the fields to exist (even when they are set to null).

I'll give this a short investigation ->  put on my todo list

Reply:
As far as I know, currently the only spot in the backend that differentiates between null and absent values when reading JSON is the `DatasetUpdateParameters` case class (it extends `TristateOptionJsonHelper` which explicitly handles this).

Reply:
Ok then this should be fine